source/tools/hubspot.rst

```diff
@@ -0,0 +1,111 @@
+Hubspot
+============
+
+What is Hubspot?
+----------------
+HubSpot is an "inbound marketing software platform" which offers features for internet marketing, such as emails, blogging and social media. This piece of software is mostly used by the marketing team.
+
+This document will mostly cover tips on how to use the emailer tool/rehabstudio email template.
+
+The rehabstudio email template
+------------------------------
+
+The 2 in 1 template allows you to switch between the event or newsletter email layouts. Each layout includes a spotlight, monthly links and footer section. The spotlight and monthly links can be hidden if not needed.
+
+Name of template:  rehabstudio_new_mailer.html
+
+Getting started on hubspot
+--------------------------
+
+Create a new email using the hubspot template
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To create a new email with Hubspot.
+
+- Click create new email.
+- Select the new template titled rehabstudio_new_mailer.html.
+- Give the email a title, then click create.
+
+The Editor
+------------------
+
+This section allows you to edit the contents of the email template. Each of the contents are stored in modules. You can manipulate the modules by showing or hiding them.
```

Comment:
@KirMcC 'each of the contents' - maybe re-word this to 'each part of the content is stored in a module'?  As long as that makes sense and is actually correct :)